payjoin/src/hpke.rs

```diff
@@ -44,6 +44,23 @@ impl HpkeKeyPair {
     pub fn public_key(&self) -> &HpkePublicKey { &self.1 }
 }
 
+fn pubkey_from_compressed_bytes(pk_bytes: &[u8]) -> Result<HpkePublicKey, HpkeError> {
+    let uncompressed_pk_bytes =
+        bitcoin::secp256k1::PublicKey::from_slice(pk_bytes)?.serialize_uncompressed();
```

Comment:
May you import `bitcoin::secp256k1` to abbreviate calls here and elsewhere in the file?

Reply:
Yes the latter `use bitcoin:secp256k1` to omit the `bitcoin::` prefix